### ISSUE TITLE
Bicycle App that allows cyclist to record and post (as an aggregate) bike rides

### ISSUE THREAD

**Issue by larrymason** [Code4Huntsville/projects_leader_board#14]
<h2>A crowdsource bicycle traffic study in real-time.

An app that allows a cyclist to use a mobile device to record in real-time bicycles rides and post is to a map (spatial-database). Such an application would help city traffic engineers and city planners better serve the biking community by getting a clear picture of where bikers are traveling and frequencies. 

Traffic engineers have long had software tools and do studies to understand automobile traffic patterns and thus they plan for automobiles. These studies and most of the software do little to account for bicycle trips and thus there is little information for knowing where to improve service, provide better lane protection and develop complete routes.

**Comment by larrymason** [comment]
This is not totally my idea, I believe the Philly Brigade is doing something akin to this. Maybe some of the code in folk-able and applicable to Huntsville. A little more research needed to look into this.

**Comment by mjcarroll** [comment]
Larry,

There are multiple applications that log this type of data.  Notably, all of these: http://alternativeto.net/software/runkeeper-pro/

Perhaps we could leverage existing apps, and give members the opportunity to "donate" their ride data.

**Comment by larrymason** [comment]
Agreed, the app is not that original... but what would be cool is to make sure that the city is aware of it so that the Transportation Department sees and uses the data... this ideas is almost as much about marketing and coding.

**Comment by chrisbeaman** [comment]
I think we should think about the solution that leverages this data, not on recording this data ourselves, because as Michael points out, there are already 20+ apps that record this data. I don't think we should market data sources to the city/make them aware of data sources because it is our job to take data sources and build solutions with them. If we come up with an idea for an app that would leverage this data, then after we've accessed/harvested this data, we can add it to our data portal, and the city can access it from there.

**Comment by larrymason** [comment]
Well I totally disagree. The biggest problem with designing good bike paths and routes in the lack of data. If we can't provide the city with data to help with planning that will certainly limit our effectiveness. But I don't want to fight about it.

**Comment by chrisbeaman** [comment]
Oh, I'm not trying to fight about it. I apologize if my response came off that way. I guess I'm disagreeing with our role as a group. We're trying to create solutions for the people, and for the city. The city already has all kinds of data that it doesn't know what to do with. We're the technologists that do stuff with data. It feels backwards to me that we would provide them with data. Wouldn't we take the data and do something with it, or build it into our open data repository?

**Comment by larrymason** [comment]
It is not true to say the city has more data than they know what to do with... all the data they have serves a specific purpose and it serves that purpose well. What we want to accomplish is help to aggregate or arrange connections between data sets which will result in new views, ways of seeing a data-driven reality. That is in general.

 As for the traffic department, they are very much data driven and they are very good at it. What they have is data about how cars, automobiles and truck use the streets... because that is the kind of data that has been collected over the years. Because that data is auto-centric they plan and build roads to accommodate cars. They don't have a lot of data as pertains to bicycle traffic. If they had more data about where bike traffic is the heaviest, where cyclist take the trips, etc., if they had more data about bicycle traffic they could do a better job of planning bike routes, and improving bike lanes etc. That is all I'm trying to say.

**Comment by larrymason** [comment]
Must have hit the wrong button... Not ready to close this tet.

**Comment by larrymason** [comment]
Code for Philly's bicycle app...a screen shot.

https://slack-files.com/files-pub/T024F66L9-F04GDL34G-f1aacfff18/screen_shot_2015-04-22_at_4.58.28_pm.png
